ldes-server-port-publication-rest/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/rest/LdesFragmentControllerTest.java

```diff
@@ -108,4 +108,17 @@ public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
                     Arguments.of("application/ld+json", Lang.JSONLD11));
         }
     }
+    @TestConfiguration
+    static class TestConfig {
+        @Bean
+        public ViewConfig viewConfig(){
+            ViewConfig viewConfig = new ViewConfig();
+            viewConfig.setShape("https://private-api.gipod.test-vlaanderen.be/api/v1/ldes/mobility-hindrances/shape");
+            viewConfig.setMemberLimit(3L);
+            viewConfig.setTimestampPath("http://www.w3.org/ns/prov#generatedAtTime");
+            viewConfig.setVersionOfPath("http://purl.org/dc/terms/isVersionOf");
+            return viewConfig;
+        }
```

Comment:
Is there no way to use the application-test.yml in this instance? or do you specifically prefer to have the testconfig inside this test class?